shared/components/shopping/shopping.component.js

```diff
@@ -144,7 +144,8 @@ class ShoppingComponent extends Panel {
   }
 
   initializeServicesSlider(){
-    let shopping = this;
+    let shopping = this,
+        total_services_api_key = shopping.apiKey('input_footprint_shopping_services_total');
 
```

Comment:
the `apiKey` method is a helper for using api key abbreviations, so you only have to pass the part beyond what's in `api_key_base`. In shopping.component.js it's

```

  get api_key_base(){
    return 'input_footprint_shopping';
  }
```

so just `apiKey('services_total')` will work.